itest/lnd_payment_test.go

```diff
@@ -504,7 +504,7 @@ func testBidirectionalAsyncPayments(ht *lntest.HarnessTest) {
 	args := []string{
 		// Increase the dust threshold to avoid the payments fail due
 		// to threshold limit reached.
-		"--dust-threshold=5000000",
+		"--dust-threshold=10000000",
```

Comment:
Is this due to the expanded definition of dust?

Reply:
yup